### PR TITLE
If available use node workerThreads

### PR DESCRIPTION
**Pull request by stefanpenner**
- [x] pending https://github.com/josdejong/workerpool/pull/52

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,7 +3,10 @@
 # Test against these versions of Node.js.
 environment:
   matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "8"
     - nodejs_version: "10"
+    - nodejs_version: "11"
```
Comment (Turbo87) [inline appveyor.yml]: do we need to test all of the versions on windows? I think we only tested latest due to CI speed reasons

Comment (stefanpenner) [reply appveyor.yml]: Well. At least 10 and 11 so we can try with and without WorkerThreads

**Comment by stefanpenner** [comment]
@Turbo87  https://ci.appveyor.com/project/embercli/broccoli-babel-transpiler/builds/22642575/job/k4rsbur9n46t2hma

**Comment by stefanpenner** [comment]
@Turbo87 any ideas?s?

**Comment by Turbo87** [comment]
@stefanpenner why are we using `npm` on AppVeyor instead of `yarn`?

**Comment by Turbo87** [comment]
also, looking at https://github.com/highlightjs/highlight.js/blob/master/CHANGES.md, the issue might have resolved itself already? 🤔

**Comment by stefanpenner** [comment]
@Turbo87 that did the trick.

**Comment by stefanpenner** [comment]
going to try and get yarn on appveyor.

**Comment by stefanpenner** [comment]
released as v7.2.0 🎉

**Pull request merged**